cirq-core/cirq/protocols/act_on_protocol.py

```diff
@@ -147,7 +150,7 @@ def act_on(
 
     arg_fallback = getattr(args, '_act_on_fallback_', None)
     if arg_fallback is not None:
-        qubits = action.qubits if is_op else qubits
+        qubits = action.qubits if isinstance(action, ops.Operation) else qubits
```

Comment:
Why the change here ?

Reply:
mypy suddenly couldn't infer that `is_op` implies that `action.qubits` existed.